flipperdevices/flipperzero-firmware
# Log MFC nonces for use with mfkey32v2

# What's new

- Adds a debug log with the nonce, nr, ar that can be used with mfkey32v2 to calculate a mifare classic key.

# Verification 

- Enable debug
- Increase log level to debug
- UID emulate mifare classic card
- open CLI and start `log`
- present to reader
- see log lines like:
```
70795 [D][MfClassic]: 939be0d5 keyA block 3 nt/nr/ar: 4e70d691 b3a576be 02c1559b
...
77521 [D][MfClassic]: 939be0d5 keyA block 3 nt/nr/ar: c6efb126 d24dd966 03fc7386
```
- run mfkey32v2 (`./mfkey32v2 939be0d5 4e70d691 b3a576be 02c1559b c6efb126 d24dd966 03fc7386`)
- Get expected key (`Found Key: [a0a1a2a3a4a5]`)

# Checklist (For Reviewer)
